docs/cli/cauldron/add/nativeapp.md

```diff
@@ -1,6 +1,6 @@
 ## `ern cauldron add nativeapp <descriptor>`
 #### Description
-* Add a new native application version to the currently activated Cauldron  
+* Add a new native application version to the currently activated Cauldron. The *complete native application descriptor* is a string with format `[nativeAppName]:[platform]:[version]` 
```

Comment:
Please make this addition in the `remarks` section of the command, not in the description of the command.
i.e add a bullet in remarks where it says `The descriptor provided to this command should be a complete native application descriptor`. No need to add the format as it is linked to the glossary